x-pack/plugin/rollup/src/main/java/org/elasticsearch/xpack/rollup/RollupResponseTranslator.java

```diff
@@ -101,7 +101,6 @@ public static SearchResponse translateResponse(MultiSearchResponse.Item[] rolled
             responses.add(item.getResponse());
         }
 
-        assert responses.size() > 0;
```

Comment:
this assertion still holds true if the assertion at the top of the method holds true.

Reply:
it looks like it's always true, I thought it's not useful, but if you think it is I can restore it.

Reply:
Yes, it's always true, but the point is to document the code like that, i.e. you don't need to think whether from that point on you always have a non-empty list of responses, the code formally tells you it is so.